ui/ui-components/pages/model/[modelId]/source/[sourceId]/schedule.tsx

```diff
@@ -497,50 +495,53 @@ export default function Page(props) {
                                       : null}
                                   </Form.Control>
                                 </Form.Group>
-                              </td>
 
-                              <td>
-                                {localFilter.key === "occurredAt" ? (
-                                  <DatePicker
-                                    selected={
-                                      localFilter.match &&
-                                      localFilter.match !== "null"
-                                        ? new Date(
-                                            parseInt(
-                                              localFilter.match.toString()
+                                {!localFilter.op
```

Comment:
The diff here misleadingly makes it look like there are big changes.  There are two changes:
(1) the `match` input is no longer housed in its own `<td>`, but shares with operation (like in the Group Rule and Property Filter UIs)
(2)  The `match` input is also now conditional -- only displays if the `op` doesn't contain the string "exist"